rocAL/rocAL/source/pipeline/node.cpp

```diff
@@ -56,16 +66,5 @@ Node::create(std::shared_ptr<Graph> graph)
 void
 Node::update_parameters()
 {
-    update_src_roi();
     update_node();
 }
-
-void
-Node::update_src_roi()
```

Comment:
Do we need not have this functions?
Are we doing roi updation in openvx / anywhere else?

Reply:
This has been removed on purpose